PWGHF/HFC/TableProducer/correlatorDsHadrons.cxx

```diff
@@ -597,11 +597,24 @@ struct HfCorrelatorDsHadrons {
         // prompt and non-prompt division
         isDsPrompt = particle.originMcGen() == RecoDecay::OriginType::Prompt;
         isDecayChan = particle.flagMcDecayChanGen() == decayChannel;
+        std::vector<int> listDaughters{};
+        std::array<int, 3> prongsId;
+        listDaughters.clear();
+        RecoDecay::getDaughters(particle, &listDaughters, std::array{0}, 1);
```

Comment:
The depth should be 2 (otherwise it checks only the first decay step, hence -> phi pi)
```suggestion
        RecoDecay::getDaughters(particle, &listDaughters, std::array{0}, 2);
```